### DISCUSSION TITLE
Swap prefix for names

### DISCUSSION THREAD

**Discussion by CelestialProvidence** [fooyin/fooyin#170] [Q&A] [answered]
I was wondering if there's a way to swap prefixes (The Chameleons > Chameleons, The) like in foobar. In foobar I had it set to $swapprefix(%\<album artist\>%) but that doesn't seem to work in fooyin.

**Comment by ludouzi** [comment, marked as answer]
$swapprefix is implemented so it should be working. Have you tried `$swapprefix(%<albumartist>%)`?
